app/src/main/java/fi/metatavu/muisti/exhibitionui/persistence/repository/LayoutRepository.kt

```diff
@@ -53,16 +54,16 @@ class LayoutRepository(private val layoutDao: LayoutDao) {
             val existing = layoutDao.findByLayoutId(id.toString())
             if (existing == null) {
                 layoutDao.insert(Layout(
-                    name = it.name,
-                    data = it.data,
+                    name = null,
```

Comment:
changing name field to null would require database migration, so use id instead of null in order to avoid that